data/models/user.py

```diff
@@ -70,6 +71,7 @@ class Sources(models.TextChoices):
     )
 
     is_dev = models.BooleanField(default="False", verbose_name="Compte développeur / technique")
+    is_elected = models.BooleanField(default="False", verbose_name="Compte élu·e")
```

Comment:
Why did you decide to add a separate field for this? The alternative I see is to determine this (maybe as a computed property?) by whether any departments are selected

Reply:
I thought conceptually they are different things. For instance, it's very likely that we will add regions as well, or EPCIs, etc. Maybe we will have a national elected official profile. We could also have elected officials without a particular geographic area for whom the homepage and actions may change to focus on stats... I preferred to have the flexibility of denoting the field independently.